File: team1/BF/jihun/Solution1.py
def solution(answers):
    guesses = [
        [1, 2, 3, 4, 5],
        [2, 1, 2, 3, 2, 4, 2, 5],
        [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]]

    cor = [0, 0, 0]
    for idx, ans in enumerate(answers):
        for i in range(3):
            if guesses[i][idx%len(guesses[i])] == ans:
                cor[i] += 1

    m = max(cor)
    answer = []
    for i, c in enumerate(cor):
        if c == m:
            answer.append(i + 1)

    return answer

File: study-2nd/level2/jiwon/solution1.js
function solution(n) {
    var answer = 0;
    
    if(n>2){
        answer = (n-2) + (n-1);
        if(answer == 3){
            answer = answer - 1;
        }else if(answer > 3){
            answer = answer - 2;
        }
    }else if(n<=2){
        answer = 1;
    }else{
        return n;
    }
       
    answer = Math.round(answer%1234567);   
    return answer;
}